src/utils/constants.ts

```diff
@@ -4,7 +4,7 @@ export const APP_ENV = process.env.REACT_APP_ENV
 export const NODE_ENV = process.env.NODE_ENV
 export const IS_PRODUCTION = APP_ENV === 'production'
 export const DEFAULT_CHAIN_ID = IS_PRODUCTION ? CHAIN_ID.ETHEREUM : CHAIN_ID.RINKEBY
-export const PUBLIC_URL = process.env.PUBLIC_URL
+export const PUBLIC_URL = process.env.REACT_APP_PUBLIC_URL
```

Comment:
`PUBLIC_URL` should not be a React environmental variable. It comes from `package.json`.

Reply:
Oh cool! I'll fix that!

Reply:
can you help me figure out how that works? who and where is PUBLIC_URL set from package.json?

Reply:
got it! this comes from package "homepage"!